### PR TITLE
fetch ABI's from sourcify if no etherscan is available or etherscan f…

### PR DESCRIPTION
**Pull request by KillariDev**
…ails

### CODE REVIEW

File: app/ts/components/Transactions.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ function formatTransactionDescription(tx: TransactionDescription) {
 		</>
 	)
 }
+const etherScanAbiKey = 'PSW8C433Q667DVEX5BCRMGNAH9FSGFZ7Q8'
```
Comment (MicahZoltu) [inline app/ts/components/Transactions.tsx:27]: Why didn't we need this before?

**Review by MicahZoltu — APPROVED** [review]
Why etherscan first instead of sourcify first?

File: app/ts/components/Transactions.tsx (hunk 1 above)
Comment (KillariDev) [reply app/ts/components/Transactions.tsx:27]: we did, it was just part of the urls instead of being a variable

**Comment by KillariDev** [comment]
> Why etherscan first instead of sourcify first?

etherscan has proxy detector and sourcify does not. So if the contract is in etherscan and its proxy, the UX is better

**Pull request merged**